google_compute_engine/ip_forwarding/tests/ip_forwarding_utils_test.py

```diff
@@ -20,25 +20,36 @@
 from google_compute_engine.test_compat import unittest
 
 
+def _create_mock_process(returncode, stdout, stderr):
+    mock_process = mock.Mock()
+    mock_process.returncode = returncode
+    mock_process.communicate.return_value = (stdout, stderr)
+    return mock_process
+
+
 class IpForwardingUtilsTest(unittest.TestCase):
 
   def setUp(self):
     self.mock_logger = mock.Mock()
     self.options = {'hello': 'world'}
-    self.mock_utils = ip_forwarding_utils.IpForwardingUtils(self.mock_logger)
+    with mock.patch(
+            'google_compute_engine.ip_forwarding.ip_forwarding_utils'
```

Comment:
Line continuations should be indented by 4 spaces. Code inside the with block should be indented by 2 spaces.